src/main/java/frc/trigon/robot/commands/LEDAutoSetupCommand.java

```diff
@@ -62,20 +52,12 @@ private Command getUpdateAutoStartPoseCommand() {
         ).ignoringDisable(true);
     }
 
-    private Color getLeftStripColor(double difference, double tolerance) {
+    private edu.wpi.first.wpilibj.util.Color getStripColor(double difference, double tolerance) {
```

Comment:
`getTargetStripColor`

Reply:
what does target color mean? It's not a position and doesn't sound right to me